geode-redis/src/test/java/org/apache/geode/redis/internal/SupportedCommandsJUnitTest.java

```diff
@@ -47,11 +48,14 @@
       "HSET",
       "HSETNX",
       "HLEN",
+      "HSCAN",
       "HMGET",
       "HSTRLEN",
       "HINCRBY",
       "HVALS",
       "HKEYS",
+      "INCRBY",
+      "INCR",
```

Comment:
Somehow other commands got moved around in this test as well (not just `HSCAN`)?  The original PR only moved `HSCAN`.

Reply:
there was a merge conflict when I did the cherry-pick originally.  I guess I didn't resolve the conflict right the firth time.  I just re-did the cherry-pick. and re-pushed.  Hopefully this is happier

Reply:
this pr didn't seem to pick up the changes (probably because I deleted the branch and re-created it).  opened the new cherry pick in a new pr https://github.com/apache/geode/pull/6127.  will close this one